classes/Utils.php

```diff
@@ -10238,4 +10238,93 @@ public static function country_options() {
 
 		return $options;
 	}
+
+	/**
+	 * Get editor list for post content.
+	 *
+	 * @since 3.0.0
+	 *
+	 * @param int $post_id post id.
+	 *
+	 * @return array
+	 */
+	public function get_editor_list( $post_id ) {
+		$editors = array();
+
+		$gutenberg_enabled = (bool) tutor_utils()->get_option( 'enable_gutenberg_course_edit' );
+		if ( $gutenberg_enabled ) {
+			$name             = 'gutenberg';
+			$editors[ $name ] = array(
+				'name'  => $name,
+				'label' => __( 'Edit with Gutenberg', 'tutor' ),
+				'link'  => add_query_arg(
+					array(
+						'post'   => $post_id,
+						'action' => 'edit',
+					),
+					get_admin_url( null, 'post.php' )
+				),
+			);
+		}
+
+		if ( is_plugin_active( 'droip/droip.php' ) ) {
+			$name             = 'droip';
+			$editors[ $name ] = array(
+				'name'  => $name,
+				'label' => __( 'Edit with Droip', 'tutor' ),
+				'link'  => add_query_arg(
+					array(
+						'action'  => 'droip',
+						'post_id' => $post_id,
+					),
+					get_permalink( $post_id )
+				),
+			);
+		}
+
+		if ( is_plugin_active( 'elementor/elementor.php' ) ) {
+			$name             = 'elementor';
+			$editors[ $name ] = array(
+				'name'  => $name,
+				'label' => __( 'Edit with Elementor', 'tutor' ),
+				'link'  => get_admin_url() . 'post.php?post=' . $post_id . '&action=elementor',
```

Comment:
Why the link value setting way is different for **Gutenberg & Element**

Reply:
Elementor, gutenberg, droip are load their editor in different URL

Reply:
Check files
https://github.com/themeum/tutor/pull/926/files